### PR TITLE
[spb] Update russian translation of "Installing Django", add "Installation (chromebook)"

### PR DESCRIPTION
**Pull request by LenaVolzhina**
Changes in this pull request:

"Installing Django" section:
- added section about requirements.txt
- updated subsection headers
- minor fixes

"Installation (chromebook)":
- translated section from scratch

**NB**: there will be several pull-requests to russian translation and we will review each others changes.

### CODE REVIEW

**Comment by magul** [comment]
Hi @LenaVolzhina! Thanks for your contribution!

Can You also apply your changes to our Crowdin project (as there's 89% coverage for current English version).

See: https://crowdin.com/project/django-girls-tutorial/ru#

**Comment by LenaVolzhina** [comment]
Hi @magul! Thank you for the quick responses to our pull-requests!

If it's acceptable we would like to merge our pull-requests here at Github at first, and then we'll do the edits to Crowdin. Our workshop here at Saint Petersburg will be held in less than a month, and we wanted to be sure all mentors have opportunity to complete updated tutorial before that.

The problem is, there is already obsolete version of russian translation at Crowdin, so in addition to our edits we will need to check that all the rest of texts are updated. Is it possible someone already made some automated or half-automated way to make this transfer? Syntax at Crowdin differs from Github, so we can't just copy-paste pieces of text.

File: ru/chromebook_setup/instructions.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Создай аккаунт на [GitHub](https://github.com).
+
+### PythonAnywhere
+
+Руководство Django Girls включает в себя раздел Разворачивание, этим словом описывается процесс,
+когда ты переносишь код, запускающий твоё новое веб-приложение, на публично доступный компьютер 
+(он называется сервер), чтобы другие люди могли видеть результаты твоей работы.
+
+Эта часть будет немного странной, если ты проходишь это руководство на Chromebook, так как 
+мы уже используем компьютер где-то в интернете (в противоположность, например, ноутбуку).
```
Comment (patiencedaur) [inline ru/chromebook_setup/instructions.md]:
```suggestion
для разработки мы уже используем удалённый компьютер где-то в интернете (а, например, не мощности своего ноутбука).
```

File: ru/chromebook_setup/instructions.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Тем не менее, будет полезно её пройти, ведь мы можем рассматривать наше рабочее пространство 
+в Cloud 9 или PaizaCloud как место для незавершенной работы, а Python Anywhere как место
```
Comment (patiencedaur) [inline ru/chromebook_setup/instructions.md]:
```suggestion
в Cloud9 или PaizaCloud как место для незавершенной работы, а PythonAnywhere как место
```

File: ru/chromebook_setup/instructions.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+### PythonAnywhere
+
+Руководство Django Girls включает в себя раздел Разворачивание, этим словом описывается процесс,
+когда ты переносишь код, запускающий твоё новое веб-приложение, на публично доступный компьютер 
+(он называется сервер), чтобы другие люди могли видеть результаты твоей работы.
+
+Эта часть будет немного странной, если ты проходишь это руководство на Chromebook, так как 
+мы уже используем компьютер где-то в интернете (в противоположность, например, ноутбуку).
+Тем не менее, будет полезно её пройти, ведь мы можем рассматривать наше рабочее пространство
```
Comment (patiencedaur) [inline ru/chromebook_setup/instructions.md]:
```suggestion
Тем не менее, будет полезно пройти раздел «Публикация», ведь мы можем рассматривать наше рабочее пространство 
```

File: ru/chromebook_setup/instructions.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+Руководство Django Girls включает в себя раздел Разворачивание, этим словом описывается процесс,
+когда ты переносишь код, запускающий твоё новое веб-приложение, на публично доступный компьютер 
+(он называется сервер), чтобы другие люди могли видеть результаты твоей работы.
+
+Эта часть будет немного странной, если ты проходишь это руководство на Chromebook, так как 
+мы уже используем компьютер где-то в интернете (в противоположность, например, ноутбуку).
+Тем не менее, будет полезно её пройти, ведь мы можем рассматривать наше рабочее пространство 
+в Cloud 9 или PaizaCloud как место для незавершенной работы, а Python Anywhere как место
+для демонстрации скорее законченных дел.
```
Comment (patiencedaur) [inline ru/chromebook_setup/instructions.md]:
```suggestion
для демонстрации законченных дел.
```

File: ru/chromebook_setup/instructions.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+Так что тебе нужно будет завести Python Anywhere аккаунт на сайте [www.pythonanywhere.com](https://www.pythonanywhere.com).
```
Comment (patiencedaur) [inline ru/chromebook_setup/instructions.md]:
```suggestion
Так что тебе нужно будет завести аккаунт на сайте [www.pythonanywhere.com](https://www.pythonanywhere.com).
```

File: ru/django_installation/instructions.md [hunk 6]
```diff
@@ -162,37 +162,57 @@ $ source myvenv/bin/activate
 (myvenv) ~$ python3 -m pip install --upgrade pip
 ```
 
-Затем запусти команду `pip install django~=1.11.0` (обрати внимание на тильду со знаком равенства: `~=`), чтобы установить Django.
+### Установка библиотек через указание требований
+
+Файл с требованиями (requirements) хранит список зависимостей, которые нужно установить с использованием
```
Comment (patiencedaur) [inline ru/django_installation/instructions.md]:
```suggestion
Файл с требованиями (requirements) хранит список зависимостей, которые нужно установить с помощью
```

File: ru/django_installation/instructions.md [hunk 7]
```diff
@@ -162,37 +162,57 @@ $ source myvenv/bin/activate
...
+`pip install`:
+
+Для начала создай файл `requirements.txt` внутри директории `djangogirls/`, используя текстовый редактор, который ты установила ранее.  Это можно сделать, создав в редакторе новый файл, а зачем сохранив его под именем `requirements.txt` в директории `djangogirls/`. После этого твоя директория будет выглядеть так:
```
Comment (patiencedaur) [inline ru/django_installation/instructions.md]:
```suggestion
Для начала создай файл `requirements.txt` внутри директории `djangogirls/`, используя текстовый редактор, который ты установила ранее. Просто создай в редакторе новый файл, а затем сохрани его под именем `requirements.txt` в директории `djangogirls/`. После этого твоя директория будет выглядеть так:
```

File: ru/chromebook_setup/instructions.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
+Если ты не используешь Chromebook, ты можешь пропустить эту секцию. 
+В противном случае процесс установки будет немного другим, и тебе нужно будет пройти только эту инструкцию по установке.
+
+
+### Облачная IDE (PaizaCloud Cloud IDE, AWS Cloud9)
+
+Облачная IDE — это инструмент, который предоставляет тебе редактор кода и доступ к компьютеру,
```
[the remark this answers is not in the document]
Comment (LenaVolzhina) [reply ru/chromebook_setup/instructions.md]: Понимаю твоё стремление расшифровать новые аббревиатуры, но мне кажется, это не добавит ясности, станет только больше непонятных слов.

**Comment by LenaVolzhina** [comment]
@patiencedaur thanks a lot for your review!

**Comment by OPersian** [comment]
@patiencedaur thanks for your review!

You might want to ask the administrators to add you to the `RU-PR` team so that you can review PRs on its behalf in future :)

**Pull request merged**